Marlin/src/gcode/config/M575.cpp

```diff
@@ -66,6 +68,7 @@ void GcodeSuite::M575() {
         if (set1) { MYSERIAL1.end(); MYSERIAL1.begin(baud); }
       #endif
 
+      if (parser.seen('U')) settings.save();
```

Comment:
Does this always happen fast enough to beat the serial disconnect and reboot cause by the baud rate change just above? I would guess, _probably_.

Reply:
Well find out when I get far enough to start testing! More work is needed to notify the host here regardless so it'll be getting changed. One worst case scenario I considered was a separate variable for serial settings that would be used in configuration store so that it could be done before the actual change. That's my fallback though if I see this fail since I don't really want to add extra variable size if I don't have to.